### PR TITLE
MAS-393 registrations are always active

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/RegistrationEdgeCasesTest.kt [hunk 1]
```diff
@@ -31,4 +33,16 @@ class RegistrationEdgeCasesTest : MockedEndpointsTestBase() {
...
+  fun `uses latest registration - two mappa registrations present`() {
+    val crn = "X445599"
+    setupNCCustodialSentence(crn)
+    setupRegistrations(registrationsResponse(), crn)
+    setupMaleOffender(crn)
+    setupNeeds(assessmentsApiNoSeverityNeedsResponse())
+    setupNoDeliusAssessment(crn)
+    calculateTierFor(crn)
+    expectTierCalculation("A2")
+  }
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/RegistrationEdgeCasesTest.kt:47]: Checks registrations are sorted with most recent first

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt [hunk 2]
```diff
@@ -350,33 +350,6 @@ internal class ProtectLevelCalculatorTest {
       validate()
     }
 
-    @Test
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:353]: this happens in the client now, by the time we get here the most recent is already the first one

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt [hunk 3]
```diff
@@ -752,32 +674,6 @@ internal class ProtectLevelCalculatorTest {
       assertThat(result.points).isEqualTo(2)
     }
 
-    @Test
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:755]: No inactive registrations any more

Comment (nickmcmahon01) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:755]: nitpick: is it worth keeping belt and braches?

**Review by nickmcmahon01 — APPROVED** [review]

**Pull request merged**